### DISCUSSION TITLE
Back camera doesn’t work on safari, Iphone 11

### DISCUSSION THREAD

**Discussion by Nintendoboi22** [community/community#139979] [GitHub Education]
I'm trying to apply to Github EDu, however I need proof that I goto a school they support(I do). However I can't upload a screenshot of my schedule, or any of the other options, and I also can't use the back camera. there is 4 options, 
- Front Camera
- Back Dual Wide Camera
- Back Ultra Wide Camera
- Back Camera
only the front camera works

**Comment by t1noo7** [comment]
It's a bit inconvenient, but perhaps choose a place with good lighting, with a divine angle so that the young man can do his best work.

**Reply by Nintendoboi22** [reply]
the camera shows up as well, nothing. when i try to take a photo to see if that fixes it, it goes back to front camera

**Comment by ebndev** [comment]
Hi @Nintendoboi22, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `GitHub Education` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁

In the meantime, please feel encouraged to explore existing discussions in the category and utilize the search in case other users may have run into the same issue!

**Comment by azinsharaf** [comment]
> I'm trying to apply to Github EDu, however I need proof that I goto a school they support(I do). However I can't upload a screenshot of my schedule, or any of the other options, and I also can't use the back camera. there is 4 options, 
> - Front Camera
> - Back Dual Wide Camera
> - Back Ultra Wide Camera
> - Back Camera
> only the front camera works

any luck? i have same problem.
